lms/templates/staff_problem_info.html

```diff
@@ -68,12 +68,12 @@
       </div>
       <div data-location="${location | h}" data-location-name="${location.name | h}">
         [
-        <a href="#" class="staff-debug-reset">${_('Reset Student Attempts')}</a>
+        <button type="button" class="staff-debug-reset">${_('Reset Student Attempts')}</button>
```

Comment:
This doesn't look so hot.

![image](https://cloud.githubusercontent.com/assets/484484/17671293/51d5da60-62e4-11e6-86d8-a89229fc9148.png)